packages/mui-material/src/IconButton/IconButton.test.js

```diff
@@ -153,7 +153,7 @@ describe('<IconButton />', () => {
       clientX: 19,
     });
     expect(getByTestId('icon-button')).toHaveComputedStyle({
-      backgroundColor: 'rgba(0, 0, 0, 0.04)',
+      backgroundColor: 'rgba(0, 0, 0, 0.039)',
```

Comment:
Rounding error on 8-bits value, but [this is not noticeable](https://jsfiddle.net/6ayd1j03/3/) for the human eye (1/1000th of luminosity change on bright colors).